language/move-core/types/src/language_storage.rs

```diff
@@ -18,13 +18,21 @@ pub const CORE_CODE_ADDRESS: AccountAddress = AccountAddress::ONE;
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Hash, Eq, Clone, PartialOrd, Ord)]
 pub enum TypeTag {
+    #[serde(rename = "bool")]
     Bool,
+    #[serde(rename = "u8")]
     U8,
+    #[serde(rename = "u64")]
     U64,
+    #[serde(rename = "u128")]
     U128,
+    #[serde(rename = "address")]
     Address,
+    #[serde(rename = "signer")]
     Signer,
+    #[serde(rename = "vector")]
     Vector(Box<TypeTag>),
+    #[serde(rename = "struct")]
```

Comment:
Hi @sblackshear  could I change this serde rename to "Struct" as well as other rename to capital case? Since "struct" is a reserved keyword in rust, the code deserialized back to RUST won't compile.